lib/install-nix.sh

```diff
@@ -31,6 +46,6 @@ fi
 echo "::add-path::/nix/var/nix/profiles/per-user/runner/profile/bin"
 echo "::add-path::/nix/var/nix/profiles/default/bin"
 
-if [[ $INPUT_NIX_PATH != "" ]]; then
+if [[ -n $INPUT_NIX_PATH ]]; then
```

Comment:
I find this harder to remember than the straightforward `!= ""`, is there any difference?

Reply:
No, it's a case of me knowing bash too well. I will simplify it.